pipelines/environments/promptflow-runtime/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: promptflow-runtime
+version: auto
+type: environment
+spec: spec.yaml
+extra_config: environment.yaml
+categories: ["PromptFlow"]
```

Comment:
Which is correct, "PromptFlow" or "prompt flow"? See #646

Reply:
updated here to follow the "prompt flow"